script.js

```diff
@@ -0,0 +1,214 @@
+<option value="3D">3D</option>
+</select>
+`
+
+const showMessage = (message, sender) => {
+  if (sender === "user") {
+    chat.innerHTML += `
+      <section class="user-msg fade-in">
+        <div class="bubble user-bubble">
+          <p>${message}</p>
+        </div>
+        <img src="assets/user.png" alt="User" />  
+      </section>
+    `
+  } else if (sender === "bot") {
+    chat.innerHTML += `
+      <section class="bot-msg fade-in">
+        <img src="assets/bot.png" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `
+  }
+  
+  setTimeout(() => {
+    document
+      .querySelectorAll(".fade-in")
+      .forEach((chatBubble) => {
+        chatBubble.classList.remove('fade-in')
+      })
+  }, 1000)
+
+  chat.scrollTop = chat.scrollHeight
+}
+
+
+//Sammanfattning
+const askForConfirmation = (selectedCathegory) => {
+showMessage(`Good to know. So you want to see some ${selectedCathegory} art, is that correct?`, "bot")
+inputWrapper.innerHTML = `
+<button id="restart">No, please redo</button>
+<button id="confirm">Yes</button>
+`
+document.getElementById("restart").addEventListener("click",() => location.reload())
+
+document.getElementById("confirm").addEventListener("click", () => {
+  inputWrapper.innerHTML = ""
+  showMessage("Yes, please take me there", "user")
+  setTimeout(() => showMessage(`Very well. Hold on and I will redirect you to the correct page in about 5 seconds. It was nice talking to you and I hope we'll talk again soon!`, "bot"),1000)
+  setTimeout(() => showMessage("3", "bot"), 5000)
+  setTimeout(() => showMessage("2", "bot"), 6000)
+  setTimeout(() => showMessage("1", "bot"), 7000)
+
+  if (selectedCathegory === "Web design") {
+  setTimeout(() => window.location.href="https://www.facebook.com/ArtbyErry", 8000)
+
+} else if (selectedCathegory === "Posters") {
+  setTimeout(() => window.location.href="https://www.illustratorcentrum.se/portfolio/erika-andersson-porath/", 8000)
+
+} else if (selectedCathegory === "Print ready") {
+  setTimeout(() => window.location.href="https://www.artstation.com/grimmulv", 8000)
+
+} else if (selectedCathegory === "Ink") {
+  setTimeout(() => window.location.href="https://www.illustratorcentrum.se/portfolio/erika-andersson-porath/", 8000)
+
+} else if (selectedCathegory === "Watercolour") {
+  setTimeout(() => window.location.href="https://www.instagram.com/artbyerry/", 8000)
+
+} else if (selectedCathegory === "Acrylic") {
+  setTimeout(() => window.location.href="https://www.facebook.com/ArtbyErry", 8000)
+
+} else if (selectedCathegory === "Digital") {
+  setTimeout(() => window.location.href="https://www.artstation.com/grimmulv", 8000)
+
+} else if (selectedCathegory === "2D") {
+  setTimeout(() => window.location.href="https://www.instagram.com/artbyerry/", 8000)
+
+} else if (selectedCathegory === "3D") {
+  setTimeout(() => window.location.href="https://www.facebook.com/ArtbyErry", 8000)
+}
+})
+}
```

Comment:
Remember the indentation, the body of the function should be indented one level, like this:
```
const askForConfirmation = (selectedCathegory) => {
  showMessage(`Good to know. So you want to see some ${selectedCathegory} art, is that correct?`, "bot")
  inputWrapper.innerHTML = `
    <button id="restart">No, please redo</button>
    <button id="confirm">Yes</button>
  `
  document.getElementById("restart").addEventListener("click",() => location.reload())

  document.getElementById("confirm").addEventListener("click", () => {
    inputWrapper.innerHTML = ""
```
PS. Closing brackets should be on the same level of indentation as the opening ones:
```
const myFunction = () => {
  const anotherFunction = () = {
    if (something) {
      /*Do this*/
    }
  }
}
```

Reply:
Another way to write conditions is switch statements. It would look something like this:
```
switch (selectedCathegory) {
  case "Web design":
    setTimeout(() => window.location.href="https://www.facebook.com/ArtbyErry", 8000)
    break
  case "Posters":
    setTimeout(() => window.location.href="https://www.illustratorcentrum.se/portfolio/erika-andersson-porath/", 8000)
    break
  /*etc*/
  }
```

Take a look [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) if you want to read more!